ui/src/app/components/genomics-extraction-table.tsx

```diff
@@ -56,6 +56,26 @@ const getIconConfigForStatus = (status: TerraJobStatus) => {
         color: colors.danger
       }
     };
+  } else if (status === TerraJobStatus.ABORTING) {
+    return {
+      icon: faSyncAlt,
+      iconTooltip: 'Aborting this extraction. This may take a few minutes.',
+      style: {
+        color: colors.warning,
+        animationName: 'spin',
```

Comment:
nit: we can reuse the spinner style from above if we pull it out into a variable